query-node/mappings/src/content/video.ts

```diff
@@ -139,7 +139,9 @@ export async function content_VideoCreated(db: DatabaseManager, event: Substrate
   })
 
   // load channel
-  const channel = await db.get(Channel, { where: { id: channelId.toString() } as FindConditions<Channel> })
+  const channel = await db.get(Channel, {
+    where: { id: channelId.toString(), relations: ['category'] } as FindConditions<Channel>,
```

Comment:
```
ERROR: Stopping the proccessor due to errors: {
  "name": "EntityColumnNotFound",
  "message": "No entity column \"relations\" was found."
```

```suggestion
    where: { id: channelId.toString() } as FindConditions<Channel>,
    relations: ['category']
```

Reply:
Sorry... I edited that recklessly. Fixed in another commit